### ISSUE TITLE
Tiny geometry

### ISSUE THREAD

**Issue by pauceano** [erwincoumans/tiny-differentiable-simulator#186]
I am working with a very simple aproach of a simulator. I am using  tds::RigidBody and  tds::RigidBody and tds::MultiBody. All the basis work well, but three are strange lacks in the geometries. I can see that some capabilities are not implemented, but I also found 'funny' things, like an apparent inconsistency in the definition of a box. The field that defines its dimensions (exent) sometimes is used as the lenght of the sides (when calculating max and min), and sometimes as the hal of it. Is anyone using this geometry? Could we make a branch to fix some basic things? I have no time to work a lot on it, but if someone is also interested maybe we can team up...

**Comment by erwincoumans** [comment]
Yes, the box isn't using extents/half extents consistently, we should clean it up to using only half extents.
